### ISSUE TITLE
Präsi

### ISSUE THREAD

**Issue by TSchiefer** [TSchiefer/MI_bigdata_project#3]
Habe schon mal ein paar Slides für die Präsentation hochgeladen. Ist nur ein Entwurf und darf gerne abgeändert werden.
Commit: 7a0043091f70c04e6af6ba895bab4c31b3a310ec

**Comment by wbuerzle** [comment]
Ich habe mir erlaubt die ersten paar Slides etwas zu überarbeiten und habe dann noch die Slides von meinem Teil hinzugefügt.

**Comment by wbuerzle** [comment]
Ps: wenn du in DataBricks eine Grafik erstellst, hat es rechts unterhalb der Grafik ein kleines Dreieck. Damit lässt sich die Grafik vergrössern und dann kann man mittels rechter Maustaste diese als Bild exportieren oder einen Screenshot machen.

**Comment by TSchiefer** [comment]
Schaut super aus! Das einzige, was ich nachfragen wollte ist, ob es absichtlich so ist, dass sowohl auf Slide 4 als auch auf Slide 5 die Original-Daten gezeigt sind. 

Mein Teil kommt dann voraussichtlich am Samstag.

**Comment by wbuerzle** [comment]
Ja. Auf Slide 4 ging es mir mehr drum wie die Daten vorliegen und was für Features zur Verfügung stehen und auf Slide 5 um die Zerlegung

**Comment by TSchiefer** [comment]
Schaffe es heute doch nicht, morgen früh aber dann sicher.

**Comment by TSchiefer** [comment]
In Commit a010e37d78f173c183c090bf9645e52b96d984e6 habe ich nun meinen Teil der Präsentation hochgeladen. Es sind nicht mehr als 2 Slides geworden. Wir sollten denke ich nochmal wegen der Aufteilung reden :)

Könntest Du die Präsentation bitte noch anschauen? Wenn alles passt, dann kann man sie umbenennen ("amazon-product-study.pptx" oder "amazon-product-study-final.pptx"). 

Laut Angabe sollten es max. 10 Slides sein. Wo können wir noch einsparen?

**Comment by wbuerzle** [comment]
Ich habe zwei Slides von mir zu einer zusammengefasst und habe bei deinen Slides noch ein bisschen Code eingeführt. Dann kann man auch kurz sagen was wir gemacht haben und nicht nur zeigen was rausgekommen ist.

**Comment by wbuerzle** [comment]
Jetzt sind es 10 Slides. Das sollte also passen
